<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonRunner</class>
 <widget class="QWidget" name="PythonRunner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1547</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Script:</string>
       </property>
       <property name="buddy">
        <cstring>te_pythonScript</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pb_scriptHelp">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="te_pythonScript">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="plainText">
        <string>
def process_bits(input, output, progress):
  output.resize(input.size())
  for i in range(0, input.size()):
    output.set(i, not input.at(i))
    
    if progress.is_cancelled():
      return
    progress.set_progress(i, input.size())</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="te_pluginOutput">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>te_pythonScript</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
